### ISSUE TITLE
underestimated SE for Gamma(link="log") ?

### ISSUE THREAD

**Issue by bbolker** [glmmTMB/glmmTMB#662]
Based on [this Stack Overflow post](https://stackoverflow.com/questions/65745148/is-there-a-difference-between-gamma-hurdle-two-part-models-and-zero-inflated-g?noredirect=1#comment116302936_65745148) it seems as though there might be a bug in the standard error calculation for Gamma models ...

**Comment by bbolker** [comment]
Bump. This seems important.  I did try a much simpler example (a Gamma with constant mean, single replicate) and it seemed fine, so we'll have to bisect ...

**Comment by mebrooks** [comment]
I agree this seems important. I have to work on lesson plans for the next 2 days. Do any of the other developers have time to check this out?

**Comment by bbolker** [comment]
Played with this a bit more (added more examples to [this code](https://github.com/glmmTMB/glmmTMB/blob/master/misc/GH662.R)) ... for an intercept-only model and an intercept-slope model the SEs seem fine. The next things I'm thinking about checking (to keep "bisecting" the differences with the example given in the original Stack Overflow post):

1. could something about the scaling of the predictor variables be messing this up? (I thought I checked scaling before but it's worth double-checking)
2. add more predictor variables? (Don't see why this would matter but ...)
3. whether there could be something wrong with [this line](https://github.com/glmmTMB/glmmTMB/blob/master/misc/GH662.R) or [this one](https://github.com/glmmTMB/glmmTMB/blob/master/glmmTMB/src/glmmTMB.cpp#L695) (missing parentheses in a `#define` macro?) (nothing jumps out at me but it seems possible, along the lines of [this SO question](https://stackoverflow.com/questions/7186504/c-macros-and-use-of-arguments-in-parentheses)
